pkg/commands/init.go

```diff
@@ -72,6 +75,20 @@ func NewInitCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
 				return err
 			}
 
+			b, err := yaml.Marshal(o.app.AppFile)
+			if err != nil {
+				return err
+			}
+			err = ioutil.WriteFile("./vela.yaml", b, 0600)
+			if err != nil {
+				return err
+			}
+			o.IOStreams.Info("\nRendering deployment config to " + color.New(color.FgCyan).Sprint("vela.yaml"))
+
+			if o.renderOnly {
```

Comment:
the renderOnly check should move before `oam.BaseRun`(line 73), or it's too late to check.

Reply:
fixed